che-editors.yaml

```diff
@@ -530,13 +530,13 @@ editors:
         volume: {}
   - schemaVersion: 2.1.0
     metadata:
-      name: che-incubator/che-idea/2020.3.4-next
+      name: che-incubator/che-idea/next
```

Comment:
for che-idea/next, maybe the tag of the image should be :next as well (and not 2020.3.4-next)

Reply:
It hasn't configured in GitHub actions yet, but it is possible to do that. We can create an issue for that and update next tags on the next iteration.

Reply:
Created the issue for that: https://github.com/eclipse/che/issues/21031